.github/workflows/certification.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         - state.memcached
         - state.mysql
         - state.sqlite
-        - bindings.alicloud.dubbo
+        - bindings.dubbo
```

Comment:
Please fix the dubbo cert test: https://github.com/dapr/components-contrib/actions/runs/4139639228/jobs/7157452071

Reply:
Please leave a suggestion comment I can just apply next time...

Reply:
Not possible until dapr/dapr is updated and then imported in cert tests. So I intend to temporarily break this test.

This is because of the local replace of contrib.

```
replace github.com/dapr/components-contrib => ../../../..
```

Note that inside the include dapr/dapr runtime the path is different and so this replacement is off by one folder.